docs/cmdline-opts/cacert.d

```diff
@@ -12,6 +12,10 @@ curl recognizes the environment variable named 'CURL_CA_BUNDLE' if it is
 set, and uses the given path as a path to a CA cert bundle. This option
 overrides that variable.
 
+With the windows version build against OpenSSL library, if cacert contain the
+string CURLOPT_CAINFO, then Windows' store of root certificates
+is used.
```

Comment:
**Instead of any other CA certificates**, right? I think it is worth emphasizing that. Possibly also add that if this is used in a build not on Windows, you'll end up with no CA certs.

Reply:
My new text:
With the windows version build against OpenSSL library, if cacert contain the
string CURLOPT_CAINFO, then Windows' store of root certificates
is used instead of any other CA certificates.
Caution: use only CURLOPT_CAINFO with curl 7.69 or later for MS-Windows using
OpenSSL library. On other configuration, curl will have no certificate.